### DISCUSSION TITLE
Welcome to V2rayU Discussions!

### DISCUSSION THREAD

**Discussion by yanue** [yanue/V2rayU#794] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LeneSamils** [comment]
各位大佬好，能教一下小弟怎么配置Trojan吗，以前没用过macos，配置上不了网啊，自建的vps
